paul-karen/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "paul-karen",
+  "version": "1.0.0",
+  "description": "Server and Express for lab 07",
+  "main": "server.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "node server.js"
+  },
+  "keywords": [
+    "npm",
+    "node",
+    "express",
+    "javascript"
+  ],
+  "author": "Paul Ritzman and Karen Lai",
+  "license": "MIT",
+  "dependencies": {
+    "express": "^4.16.3",
+    "npm": "^6.0.1"
```

Comment:
You probably don't actually have a dependency on npm; did you accidentally run `npm install npm` in this project?